tests/e2e/tests/pilot/egress_rules_test.go

```diff
@@ -59,7 +59,7 @@ func TestEgress(t *testing.T) {
 			// Note that we're using http (not https). We're relying on Envoy to convert the outbound call to
 			// TLS for us. This is currently the suggested way for the application to call an external TLS service.\
 			// If the application uses TLS, then no metrics will be collected for the request.
-			url:               "http://httpbin.org:443/headers",
+			url:               "http://www.httpbin.org:443/headers",
```

Comment:
can't you use https and remove the 443?

Reply:
You have to use HTTP to port 443 for Istio to handle it. Istio will perform TLS origination. Otherwise, the traffic is encrypted and the URL/header cannot be inspected by Istio. So Istio cannot perform filtering based on URLs when the traffic is originated as HTTPS.

Please see an explanation here https://istio.io/blog/2018/egress-https.html#tls-origination-by-istio